business_requirement_deliverable_default/models/business_requirement_deliverable_default.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resouce_lines(self):
+        rl_data = self.product_id.resource_lines.copy_data()
+        rl_data = [(0, 0) + (x,) for x in rl_data]
```

Comment:
@duanyp1991 According to functionality we had discuss yesterday I changed it by "Update" to add new resource_lines without duplicate or delete previous resources.
`[(1, ids[index], item) for index, item in enumerate(rl_data)]`